lib/saml_idp/incoming_metadata.rb

```diff
@@ -63,6 +63,15 @@ def contact_person
     end
     hashable :contact_person
 
+    def unspecified_certificate
```

Comment:
One example of SAML 2.0 spec it has too many optional fields for a too generic solution.
Gem missing if there certificate without specifying usage (`@use='signing'`) of the certificate. For this reason, we need to at least make it retrievable for application.